docs/pipelines/agents/includes/v3/prepare-permissions.md

```diff
@@ -11,11 +11,11 @@ ms.date: 09/22/2023
 
 The user configuring the agent needs pool admin permissions, but the user running the agent does not.
 
-The folders controlled by the agent should be restricted to as few users as possible and they contain secrets that could be decrypted or exfiltrated.
+The folders controlled by the agent should be restricted to as few users as possible as they contain secrets that could be decrypted or exfiltrated.
```

Comment:
```suggestion
The folders controlled by the agent should be restricted to as few users as possible because they contain secrets that could be decrypted or exfiltrated.
```